R/zchunk_L123.bcoc_awb_R_S_T_Y.R

```diff
@@ -1,24 +1,24 @@
 #' module_emissions_L123.bcoc_awb_R_S_T_Y
 #'
-#' Briefly describe what this chunk does.
+#' Produces BC and OC AWB emission factors for each Ag production technology.
```

Comment:
Good point! Updated to define that. (Is agricultural waste burning on fields (AWB). This is specific burning of waste on fields, as distinguished from use of ag wastes for cooking or heating, which GCAM lumps into one type of biomass or another.)